src/apis.json

```diff
@@ -7,13 +7,27 @@
   {
     "name": "fs",
     "description": "The fs or filesystem API allows you to create, read, and modify files on the repl's filesystem.",
-    "typeNames": ["WatchFileWatchers", "WatchTextFileWatchers"],
+    "typeNames": [
+      "WatchFileWatchers",
+      "WatchTextFileWatchers",
+      "WatchFileWatcherOnChange",
+      "WatchFileWatcherOnError",
+      "WatchFileWatcherOnMoveOrDelete",
+      "WatchTextFileWatcherOnReady",
+      "WatchTextFileWatcherOnChange",
+      "WatchTextFileWatcherOnError",
+      "WatchTextFileWatcherOnMoveOrDelete",
+      "FsNode",
+      "OnMoveOrDeleteArgs",
+      "TextFileWatcherReadyArgs",
+      "TextFileWatcherOnChangeArgs"
+    ],
```

Comment:
side note: not a fan of arguments showing up some distance away from the function definitions. we should systematically fix this, and not need to add to this typeName array every time we add functions.

approving since this is strictly better than what we have for now, but just giving you something to think about!

Reply:
Noted.  I'll try to fix this in the future once the docs get pretty big.